test/e2e/build_test.go

```diff
@@ -456,6 +456,33 @@ RUN find /test`, ALPINE)
 		Expect(session.OutputToString()).To(ContainSubstring("/test/dummy"))
 	})
 
+	It("podman remote build must not allow symlink for ignore files", func() {
```

Comment:
By a curious coincidence, I've spent the morning working on this very test. As written, this test is completely useless. If for any reason you need to repush, like to disable the failing containerized test, you might want to cherrypick #20582